vignettes/region_of_practical_equivalence.Rmd

```diff
@@ -120,7 +120,7 @@ p_direction(model)
 rope(model, ci=1)
 ```
 
-As you can see, by simply dividing the predictory by 100, we **drastically** changed (*we divided it by 100*) the conclusion related to the **percentage in ROPE** (which became very close to `0`): the effect could now be **interpreted as being significant**. Thus, we recommend, when reporting or reading ROPE results, to pay close attention to the unit of the predictors.
+As you can see, by simply dividing the predictor by 100, we **drastically** changed the conclusion related to the **percentage in ROPE** (which became very close to `0`): the effect could now be **interpreted as being significant**. Thus, we recommend paying close attention to the unit of the predictors when selecting the ROPE range (e.g., what coefficient would correspond to a small effect?), and when reporting or reading ROPE results.
```

Comment:
in this case, it is not particularly the ROPE range (which is fixed in both cases and depends on the unit of the outcome) but the unit of the predictor

Reply:
Yes, that's kind of the point I think I'm trying to make - the ROPE range should correspond to the predictors' units. How would you change this?

Reply:
Not sure, but maybe my quote here helps: https://github.com/easystats/bayestestR/issues/144#issuecomment-495274448